pulpcore/cli/file/distribution.py

```diff
@@ -71,6 +71,12 @@ def distribution(ctx: click.Context, pulp_ctx: PulpCLIContext, distribution_type
     repository_option,
     content_guard_option,
     pulp_labels_option,
+    click.option(
+        "--checkpoint",
+        is_flag=True,
+        default=False,
+        help=_("Create a checkpoint distribution."),
+    ),
```

Comment:
I think this should be tri-state: Set, unset, don't touch.
Would this work?
```suggestion
    click.option(
        "--checkpoint/--not-checkpoint",
        #  is_flag=True,  # this is implied by the "/" i think
        default=None,
        help=_("Create a checkpoint distribution."),
    ),
```